rust/services/call/host/src/host/error/preflight.rs

```diff
@@ -92,11 +89,11 @@ impl From<GuestTransactError> for TransactError {
 
 #[derive(Debug, Error, PartialEq)]
 pub enum RevertError {
-    #[error("revert: <empty>")]
+    #[error("Transaction reverted: <empty>")]
     EmptyRevert,
-    #[error("revert: {0:?}")]
+    #[error("Transaction reverted: {0:?}")]
     Revert(GenericRevertReason),
-    #[error("raw bytes revert: {0}")]
+    #[error("Transaction reverted with non-UTF-8 bytes: {0}")]
```

Comment:
What does it mean it got reverted *with* non-UTF-8 bytes? Does it mean it had wrong encoding, or something else?

Reply:
EVM revert function accepts raw bytes and EVM interface returns raw bytes.
Usually folks make use of it through Solidity by utilizing `revert("message")`. Then we can decode it.
Sometimes people use raw assembly or other EVM languages and then they can pass some bytes that are not valid UTF-8. This variant covers that